platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -75,6 +75,9 @@ android {
             testCoverageEnabled true
             jniDebuggable true
         }
+        qa {
```

Comment:
Could you rename this build type to `monkey`? My idea is that things like the strict mode should still be enabled for a manual qa.